netops_demo/py/functions/generate/generate.py

```diff
@@ -89,14 +87,31 @@ def _stop(context):
     # go back to idle
     context.user_data.state = 'idle'
 
-
-def _generate(context, start_timestamp=None, end_timestamp=None, interval=None, max_samples_per_batch=None, target=None):
-
+def _locations(context):
+    deployment = context.user_data.deployment
+    context.logger.info_with('Sending list of locations')
+
+    locations = []
+    for company in deployment.companies:
+        for i, location in enumerate(company.locations):
+            current_location = {
+                'key': f'{company.name}/{i}',
+                'latitude': location[0],
+                'longtitude': location[1],
+                'name': f'{company.name}/{i}'
+            }
+            locations.append(current_location)
+
+    return locations
+
+def _generate(context, start_timestamp=None, end_timestamp=None, interval=None, max_samples_per_batch=None,
```

Comment:
pep8: each argument in its own line or all in one line

Reply:
IDE Formatting, Done